src/App.css

```diff
@@ -4,8 +4,9 @@ body{
   color: whitesmoke;
   font-family: fantasy;
   padding: 0;
-  margin: 0;
-  width: 100%;  
+  margin: 0;/*
+  width: 100vw;  
+  height: 100vh;*/
```

Comment:
make sure you go back through your code at some point and delete stale comments like this one.